.kitchen.yml

```diff
@@ -31,6 +31,15 @@ platforms:
   - name: windows-2012r2
     driver_config:
       box: chef/windows-server-2012r2-standard
+# Solaris 10 requires the OS ISO to be mounted into `/cdrom` in order for the
```

Comment:
There's a 10 and 11 script in the bento project. I updated the 11 box we test with in the last year